app/assets/stylesheets/utilities.scss

```diff
@@ -362,3 +362,15 @@ span.spoiler {
 .nowrap {
   white-space: nowrap;
 }
+
+@each $side in $sides {
+  .border-#{$side}-none {
+    border-#{$side}: none;
+  }
+}
+
+@each $align in $text-aligns {
+  .text-#{$align} {
+    text-align: $align;
+  }
+}
```

Comment:
Likewise, these feel like changes that should be part of Co-Design. Perhaps not work for this PR, but we should log the missing utilities so that they can be added later.

Reply:
`text-(left|right|center)` can probably go - I missed that the text alignment utils are described in the [typography section](https://design.codidact.org/utilities/typography/) of Co-Design (that said, I heavily prefer shorter WindiCSS/UnoCSS style `text-center` and the like to `has-text-align-center`, which is extremely verbose for what it does [but that's an overall issue with our utilities that needs to be discussed separately])